ci/properties/generator-container-ossf-slsa3-publish.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "SLSA Container generator",
+  "creator": "Open Source Security Foundation (OpenSSF)",
+  "description": "Generate SLSA3 provenance for your Docker container images",
+  "iconName": "generator-container-ossf-slsa3-publish",
```

Comment:
`generator-generic-ossf-slsa3-publish` is a valid name. The above icon doesn't exist